### DISCUSSION TITLE
port metrics hiding from Pinafore

### DISCUSSION THREAD

**Discussion by aescling** [cat-KIBI-glitch/mastodon#15] [Ideas]
[Pinafore][pinafore] includes several features which purportedly reduce the addictive qualities of social media; see the settings [here][wellness].

for us, there is not an acessibility need here, but we **much** prefer using mastodon with hidden metrics, and would love to reimplement metrics hiding in the glitch edition front end (and perhaps hide metrics as a default setting?).

the remaining settings seem worth porting over, and don’t *seem* like they ought to be intrusive or difficult to implement. *if* we should port greyscale mode (very unimportant to me), it would be best as a flavour i think.

[pinafore]: https://github.com/nolanlawson/pinafore
[wellness]: https://pinafore.social/settings/wellness

**Comment by marrus-sh** [comment]
Do you mean boost/favourite metrics? I’m definitely fine hiding those.

I believe hiding followers/following metrics is already implemented; honestly I think we might want to revisit that to make it not as all‐or‐nothing though.

**Reply by aescling** [reply]
yeah i actually like showing follower/following numbers  because (i’m sorry if this is a bit cynical but) it can be useful to know if somebody has a lot of clout or not, at the very least to know to expect shit if you have to start shit. (and also, i usually don't follow large accounts as a  rule lol, so i would personally like to be able to see the counts if i were thinking about following somebody)

my **personal** preferences on these things would be that, *by default*,

* don't show boost, fave, reply counts, even when you visit the specific status
* do show follower/following counts if you visit the profile
* don't show notification indicators (not to mention counts) either in the app or on the favicon

and ofc this would all be configurable

…uh, anyway! what exactly do you mean about not making the follower/following display all or nothing, exactly?

**Reply by marrus-sh** [reply]
in past discussions what we’ve generally thought was that follower/following numbers should be available but rounded (so it’s not obvious if you gained/lost a follower, just what the rough order of magnitude of followers is). i don’t think anybody ever implemented something like that, but i could probably dig up the Issues where those discussions took place.

i think similar arguments could be made with boosts and favs; knowing their rough values might be useful to somebody, but exact values are pretty much only harmful.
